code/__DEFINES/mob.dm

```diff
@@ -10,3 +10,9 @@
 #define MOB_HOLOGRAPHIC 16 //holocarps
 #define MOB_SUPERNATURAL 32
 #define MOB_NO_PETRIFY 64 //can't get petrified
+
+// For blob_act
+
+#define WEAK 3
```

Comment:
Should be named BLOB_ACT_WEAK, both for clarity and to avoid #define conflicts in the future (there's no compiler warning for overriding defines, so there will be issues if somebody else defines WEAK)

Reply:
Still applies

Reply:
Good point